DataFormats/MuonReco/src/classes_def.xml

```diff
@@ -42,11 +42,16 @@
   <class name="reco::MuonEnergy" ClassVersion="10">
    <version ClassVersion="10" checksum="1493325087"/>
   </class>
-  <class name="reco::MuonChamberMatch" ClassVersion="11">
+  <class name="reco::MuonChamberMatch" ClassVersion="15">
```

Comment:
please do not introduce intermediate versions that you had while developing the code.
Since 11 was the last one, drop the current 12,13,14 and rename the current 15 to be 12.